src/structures/Message.js

```diff
@@ -238,7 +259,7 @@ class Message extends Base {
      * Flags that are applied to the message
      * @type {Readonly<MessageFlags>}
      */
-    this.flags = new MessageFlags(data.flags).freeze();
+    this.flags = new MessageFlags(partial ? data.flags ?? 0 : data.flags).freeze();
```

Comment:
Isn't this technically wrong? What if a message has flags, we get a MESSAGE_UPDATE for god knows what, and the flags don't change? Won't this make the cached flags invalid with what they actually are?

Reply:
This seems to be errored logic in the original as well, I'll replace 0 with this.flags